### PR TITLE
Add initial helm api docs

### PR DESCRIPTION
**Pull request by akashshinde**
Add helm related API docs.

following APIs are documented
1. /api/helm/releases
2. /api/helm/release
3. /api/helm/template

### CODE REVIEW

File: docs/helm_apis.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+**List Helm Releases**
```
Comment (benjaminapetersen) [inline docs/helm_apis.md]: I'm good with a `/docs` dir, might suggest you make it a `/docs/helm` dir, and make this your main `README.md` file inside.

Comment (akashshinde) [reply docs/helm_apis.md]: Done

**Comment by akashshinde** [comment]
cc @benjaminapetersen, please take a look now

**Comment by benjaminapetersen** [comment]
/lgtm

but prob won't merge due to bugzilla block.

**Comment by benjaminapetersen** [comment]
/retest

```
could not wait for build: the build src failed after 41s with reason PushImageToRegistryFailed: Failed to push the image to the registry.
```

**Pull request merged**